src/license_expression/__init__.py

```diff
@@ -116,6 +126,49 @@ class ExpressionParseError(ParseError, ExpressionError):
 ).finditer
 
 
+class ExpressionInfo:
+    """
+    The ExpressionInfo class is returned by Licensing.validate() where it stores
+    information about a given license expression passed into
+    Licensing.validate().
+
+    The ExpressionInfo class has the following fields:
+    - original_license_expression: str.
+      - This is the license expression that was originally passed into Licensing.validate()
+    - normalized_license_expression: str.
+      - If a valid license expression has been passed into `validate()`,
+        then the license expression string will be set in this field.
+    - errors: list
+      - If there were errors validating a license expression,
+        the error messages will be appended here.
+    - valid_symbols: list
+      - If a valid license expression has been passed into `validate()`,
+        then the license symbols from the license expression will be
+        appended here.
+    - valid_exception_symbols: list
+      - If a license symbol in the license expression is a license exception,
+        then that license symbol will be appended here.
+    - invalid_symbols: list
+      - If an invalid license expression has been passed into `validate()`,
+        then the invalid license symbols from the license expression will be
+        appended here.
+    """
+    def __init__(
```

Comment:
nit: the args below should be indented Black-style

Reply:
Done.